code/espurna/scheduler.cpp

```diff
@@ -60,6 +62,96 @@ constexpr bool restoreLast() {
 
 // -----------------------------------------------------------------------------
 
+#if API_SUPPORT
+
+// UTILS
+
+template <typename V = const char*>
+bool _setJsonVariant(const SettingsKey& k, const JsonVariant& v) {
+    if (!v.is<V>()) {
+        return false;
+    }
+    return setSetting(k, std::move(v.as<V>()));
```

Comment:
move is redundant, v.as<V>() already becomes either T&& or const T& depending on the function